stag-library-compiler/src/main/java/com/vimeo/stag/processor/generators/StagGenerator.java

```diff
@@ -299,8 +299,14 @@ private TypeSpec getAdapterFactorySpec() {
                 .addParameter(Gson.class, "gson")
                 .addParameter(ParameterizedTypeName.get(ClassName.get(TypeToken.class), genericTypeName),
                               "type")
+                .addStatement("setOrThrow(gson)")
```

Comment:
There is a license JavaDoc in this file. But, it is not in other files. Should it be removed?